recipes-kernel/linux/linux-pinenote-smaeul_5.16-rc8.bb

```diff
@@ -0,0 +1,25 @@
+SUMMARY = "Linux kernel customized for the PineNote A55 by smaeul"
```

Comment:
Do we need three kernel recipes?

Reply:
I think it would be best to just pick one now and we can update it in the future if it changes. Otherwise it's confusing for users which one to use

Reply:
ok will do